### PR TITLE
Update docs to show model: param usage

### PR DESCRIPTION
**Pull request by tdonia**
Feel free to edit content/tone as you like -- just wanted to include some pointers in case anyone else ever needs this param.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,20 @@ EasyTranslate.detect "This is definitely English!", :confidence => true # => { :
 EasyTranslate.detect ['This is definitely English!', '我姓譚'], :confidence => true # => [{ :language => 'en', :confidence => 0.77272725 }, { :language => 'zh-CN', :confidence => 1.0 }]
 ```
 
+## NMT? PBMT? I want a neural network!
```
Comment (seejohnrun) [inline README.md]: lol

File: README.md [hunk 2]
```diff
@@ -75,7 +75,20 @@ EasyTranslate.detect "This is definitely English!", :confidence => true # => { :
...
+Google Translate now replaces Phrase Based Machine Translation (PBMT) with Neural Machine Translation (NMT) automatically where possible. If you prefer PBMT or need to compare the results, you can use the `model:` parameter with either `nmt` or `base` to force the model selection:
 
+```ruby
+2.2.5 :019 > EasyTranslate.translate("El cuervo americano es un ave con plumas negras iridiscentes sobre todo su cuerpo.", from: "es", to: "en", model: "nmt")
```
Comment (seejohnrun) [inline README.md]: can we remove the version information and irb prompt here?

File: README.md [hunk 3]
```diff
@@ -75,7 +75,20 @@ EasyTranslate.detect "This is definitely English!", :confidence => true # => { :
...
+ => "The American Raven is a bird with iridescent black feathers over its entire body."
```
Comment (seejohnrun) [inline README.md]: I commonly put these inside a comment just for the syntax highlighter to be chill, but obviously optional

**Comment by tdonia** [comment]
updated!

**Comment by seejohnrun** [comment]
Sweet, thanks @tdonia

**Pull request merged**